script.js

```diff
@@ -1,12 +1,42 @@
-let colorlist = ['gold', 'yellow', 'turquoise', 'red']
+let colorlist = ['gold', 'black', 'turquoise', 'red'];
+let img;
+let fun = ['dance', 'play', 'drink', 'sleep', 'fun', 'work', 'school', 'love'];
+let food = ['pizza', 'burgers', 'tacos', 'ramen', 'seafood', 'curry', 'wings'];
+let drink = ['beer', 'boba', 'coffee', 'tea', 'liquor', 'soda', 'water'];
+let rand;
 
 function setup() {
   createCanvas(windowWidth, windowHeight);
-    background(255);
+  background(255);
+  textSize(40);
+  image(img, 500, 200);
+  rand = (Math.floor(random() * 3));
+}
+
+function preload() {
+  img = loadImage('./unnamed.png');
```

Comment:
In the future, you can name your image files and variables to be more descriptive. e.g. `bulbImg = loadImage('./light_bulb.png')`